### ISSUE TITLE
Write debug build commands in a generalized way (without any specific version number)

### ISSUE THREAD

**Issue by eemihauk** [TheSoftwareFactory/lokki-android#24]
The documentation fix #23 is valid for just the current version of the app. There should be a command written without any specific version number, e.g.
```
$ adb install -r App/build/outputs/apk/lokki-v[VERSION_NUMBER]-debug.apk
```
Maybe then also leave an example with a concrete version number?
It would also probably be good to add how to find the version number of the build.

**Comment by akiratk0355** [comment]
It would be better. And also adding a link to the latest release would make it easy. For example:

Check the latest release version number [here](https://github.com/TheSoftwareFactory/lokki-android/releases)

Then we don't need to refresh version info anymore.

**Comment by eemihauk** [comment]
Yes, I just added the releases link to the main lokki repo [wiki](https://github.com/TheSoftwareFactory/lokki/wiki#deployment). Adding it to the lokki-android repo README would also be good.

**Comment by eemihauk** [comment]
Solved in #25
